_posts/2021-01-09-Julia-1.6-what-has-changed-since-1.0.md

```diff
@@ -0,0 +1,639 @@
+---
+title: "Julia 1.6: what has changed since Julia 1.0?"
+layout: default
+tags:
+   - julia
+---
+
+Julia 1.0 came out well over 2 year ago.
+Since then a lot has changed; and a lot hasn't.
+Since Julia 1.0 was a commitment to no breaking changes;
+but that is not to say no new features have been added to the language.
+
+<!--more-->
+
+Julia 1.6 is the first feature release since 1.0.
+Prior to 1.0 all releases were feature releases; they came out when they were ready.
+Since then all releases (except 1.6) have been timed releases.
+The release candidate is cut from the main branch every 3 months; and they are released after an [extensive round of additional checking](https://github.com/JuliaLang/julia/pull/35846), which often takes weeks to months.
+Julia 1.6 was soft-slated to be the new long-term-support (LTS) version that would have bug-fixes backported to it for the next few years.
+The current LTS is julia 1.0, which has now had 5 patch releases made.
```

Comment:
```suggestion
The current LTS is Julia 1.0, which has now had 5 patch releases made.
```

I dunno how strict you like to be with capitalization of Julia when referring to it as a language (rather than the `julia` binary, but I'm gonna keep highlighting them anyways